app/config/default.json

```diff
@@ -1,6 +1,10 @@
 {
   "frontend": {
-    "apiPath": "api/v1"
+    "apiPath": "api/v1",
+    "oidc": {
+      "authority": "https://dev.loginproxy.gov.bc.ca/auth/realms/standard",
+      "clientId": "bc-box-local-4545"
+    }
```

Comment:
Please revert this from history. There is no business having these default configurables appear in the codebase.